### PR TITLE
Update Get friend and get game commands

### LINKED ISSUE #55: Update 'get' command
- Refactor existing `get` command into separate `friend --get`
- Add `game --get` command

### PR DESCRIPTION
**Pull request by YeluriKetan**
# Changes
- Refactor friend get command and appropriate parser
- Add game get command and appropriate parser
- Add tests for the necessary classes

# Issues
Resolves #55 

# Testing
Most of commands are now working as intended.
- There is a exception being thrown from the UI when the `link` command is invoked.
- UI needs to be updated for the result of the get command.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/GetFriendCommandTest.java [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
+//        String friendId = "98765432";
+//        String expectedMessage = String.format(GetFriendCommand.MESSAGE_FRIEND_FULL_INFORMATION, friendId);
+//        FriendIdMatchesKeywordPredicate predicate = new FriendIdMatchesKeywordPredicate(friendId);
+//        GetFriendCommand command = new GetFriendCommand(predicate);
+//
+//        expectedModel.updateFilteredFriendsList(predicate);
+//        assertCommandSuccess(command, model, expectedMessage, expectedModel);
+//        assertEquals(List.of(BENSON), model.getFilteredFriendsList());
+//    }
+//}
```
Comment (MarcusTXK) [inline src/test/java/seedu/address/logic/commands/GetFriendCommandTest.java:80]: Consider deleting this file if it is no longer used

File: src/test/java/seedu/address/logic/parser/GetFriendCommandParserTest.java [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+//        GetFriendCommand expectedGetFriendCommand =
+//                new GetFriendCommand(new FriendIdMatchesKeywordPredicate(friendId));
+//        assertParseSuccess(parser, " " + FLAG_FRIEND_SPACE + friendId, expectedGetFriendCommand);
+//
+//        // leading, within and trailing whitespaces
+//        assertParseSuccess(parser, "    " + FLAG_FRIEND_SPACE + "    " + friendId
+//                + "    \t", expectedGetFriendCommand);
+//    }
+//
+//}
```
Comment (MarcusTXK) [inline src/test/java/seedu/address/logic/parser/GetFriendCommandParserTest.java:49]: Consider deleting this file as well

**Review by MarcusTXK — APPROVED** [review]
LGTM, good test coverage for newly made commands. Consider deleting the commented out files as they are no longer needed.

File: src/test/java/seedu/address/logic/commands/GetFriendCommandTest.java (hunk 1 above)
Comment (YeluriKetan) [reply src/test/java/seedu/address/logic/commands/GetFriendCommandTest.java]: Good catch. This was supposed to be a temporary fix when I was writing new and improved tests, and were supposed to be deleted after that. Let me fix it.

**Pull request merged**